doc/sphinx/azhelpgen/azhelpgen.py

```diff
@@ -10,10 +10,10 @@
 from sphinx.util.compat import Directive
 from sphinx.util.nodes import nested_parse_with_titles
 
-from azure.cli.application import Application, Configuration
+from azure.cli.application import APPLICATION, Configuration
 import azure.cli._help as _help
 
-app = Application(Configuration([]))
+app = APPLICATION
```

Comment:
this isn't in the product (this script is separate) so all other scripts should be unaffected